donut/modules/courses/templates/scheduler.html

```diff
@@ -461,7 +486,58 @@ <h1>Course Scheduler</h1>
     function displayUnits() {
       units.text('Units: ' + String(coursesUnits))
     }
-
+  $('#modal').on('show.bs.modal', function(e) {
+    var courseNum = $(e.relatedTarget).data("courseNum");
+    var courseId = $(e.relatedTarget).data("courseId");
+    displayNotes(courseNum, courseId); 
+  });
+  $('#editButton').click(function(e) {
+    $('#postNotes').show();
+    $('#viewNotes').hide();
+    $('#editButton').hide();
+  });
+  $('#postNotes').submit(function(e) {
+    e.preventDefault();
+    $.ajax({
+      url: '/1/scheduler/edit_notes',
+      type: 'POST',
+      data:$('#postNotes').serialize(),
+      success:function(){
+        displayNotes($('#courseNum').val(), $('#courseId').val());
+      },
+      error: function() {
+        displayWarning('Failed to save notes')
+      }
+    });
+  });
+  function displayNotes(courseNum, courseId) {
```

Comment:
Just clarifying—are notes hidden until you click the section on the scheduler?

Reply:
Yes